courts_db/data/courts.json

```diff
@@ -2541,6 +2541,37 @@
         "system": "special",
         "type": "trial"
     },
+    {
+        "citation_string": "Cal. State Bar Ct.",
+        "court_url": "https://www.statebarcourt.ca.gov/",
+        "dates": [
+            {
+                "end": null,
+                "start": "1989-01-01"
+            }
+        ],
+        "examples": [
+            "State Bar Court of California",
+            "California State Bar Court",
+            "State Bar Court Review Department",
+            "State Bar Court Hearing Department"
+        ],
+        "id": "calstatebar",
+        "level": "special",
```

Comment:
I know there are may be some inconsistencies in courts-db but I think level should be `ljc` limited jurisdiction court?